scripts/publish_for_desktop

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+yarn && yarn build
+rm -rf ../toolkit-for-ynab-gh-pages/desktop-updates
+mkdir ../toolkit-for-ynab-gh-pages/desktop-updates
+cp dist/manifest.json ../toolkit-for-ynab-gh-pages/desktop-updates/
+echo "Hi Kevin, I'm not sure what to do for the zip file..."
```

Comment:
I have another directory on my build machine that's always on the `gh-pages` branch. It just copies the zip file over there, and then I commit and push it.

That being said, desktop can't actually use the file anyway, so I'm not sure why I've bothered maintaining it all this time. Now that we're web extensions it should be much easier to load it in as before I was trying to shim Kango. Maybe it's time to dust off that project.